quickstarts/Tuning.ipynb

```diff
@@ -48,7 +48,7 @@
       "source": [
```

Comment:
<p>We should avoid having commented-out code checked in (except in very specific cases) as we don't know if it stops working.</p><p> </p><p>Un-comment everything here, and isolate the branching point in a single clear line.</p><p> </p><p>e.g.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">dict_data = [{...}, ...]
...
csv_file = 'data.csv'
...

# Here you can specify any of the supported formats, e.g. csv_file or dict_data
training_data = csv_file
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Done